Doc/library/fcntl.rst

```diff
@@ -18,6 +18,8 @@ interface to the :c:func:`fcntl` and :c:func:`ioctl` Unix routines.  For a
 complete description of these calls, see :manpage:`fcntl(2)` and
 :manpage:`ioctl(2)` Unix manual pages.
 
+.. include:: ../includes/wasm-notavail.rst
```

Comment:
Wouldn't it be better to put this after the introduction prose and right above the "``versionchanged`` annotations, which is where it typically goes inside of functions and elsewhere? Its confusing to me to have it stuck right in the middle of the introduction paragraphs.

Reply:
IMO it is fine to have a short paragraph to introduce the module, then the availability note.
Is this the only doc with that layout?